templates/projects/annotations.html

```diff
@@ -0,0 +1,26 @@
+{% extends "projects/base.html" %}
+{% set active_page = "projects" %}
+{% set active_link  = "settings" %}
+{% set section = _('Configure Annotation Parameters') %}
+
+{% import "projects/_helpers.html" as helper %}
+{% from "_formhelpers.html" import render_field, render_checkbox_field, render_select2_field %}
+
+{% block projectcontent %}
+<form method="post" action="{{ url_for('project.annotation_config', short_name=project.short_name) }}">
+    <fieldset>
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+        {{ render_field(form.dataset_description, class_="input-xlarge", placeholder=_('A short description of the type of data(set) that will be annotated along with representative examples. (e.g., The data annotated in this project are Chinese BN news articles.)')) }}
+        {{ render_field(form.provider) }}
+        {{ render_field(form.restrictions_and_permissioning, class_="input-xlarge", placeholder=_('A complete list of relevant security-related properties and requirements of the dataset (e.g., Original data is stored in the Data DMZ, behind PVF, and encrypted)')) }}
```

Comment:
Do we want to expose the internal terminology or can we use something more generic?

Reply:
yes, generic terminology makes sense.